documentation/asciidoc/accessories/iqaudio/configuration.adoc

```diff
@@ -1,12 +1,12 @@
 == Configuration
 
-A pre-programmed EEPROM (Electrically Erasable Programmable Read-only Memory) is included on all Raspberry Pi audio boards. By doing so, Raspberry Pi OS is able to automatically detect and configure itself, enabling Raspberry Pi audio boards to be plug-and-play. Using Raspberry Pi OS (32-bit) with the Desktop environment and your board attached, you can utilise the built in GUI (Graphical User Interface) by right clicking on the audio settings in the top right hand corner of your screen, which allows you to switch between the on-board audio settings and the HAT audio settings:
+A pre-programmed EEPROM is included on all Raspberry Pi audio boards. Raspberry Pi audio boards are designed to be plug-and-play; Raspberry Pi OS is able to automatically detect and configure itself. In Raspberry Pi OS, right-clicking on the audio settings in the top right-hand corner of your screen will allow you to switch between the on-board audio settings and the HAT audio settings:
```

Comment:
Only a minor quibble, but perhaps the latter part of "Raspberry Pi audio boards are designed to be plug-and-play; Raspberry Pi OS is able to automatically detect and configure itself." could be reworded to "Raspberry Pi OS is able to automatically detect them and configure itself appropriately." as otherwise it might not be clear what Raspberry Pi OS is detecting? :shrug: